### PR TITLE
Restructured LookupParticipantIdModal to Remove Redux Alerts

### PR DESCRIPTION
**Pull request by jcq**
Connects #13615

### Description
This restructures the `LookupParticipantIdModal` component to remove Redux connections and re-implement success/error `Alert` within this component rather than `QueueFlowModal` (from which they were removed).

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] Alerts show within `LookupParticipantIdModal`
- [ ] Alerts do not show on Team Management page behind modal
- [ ] Redux-connected success Alerts do not show in other components that rely on `QueueFlowModal`

### Testing Plan
1. Go to "Caseflow Team Management"
2. Scroll down to "Private Bar" section and click "Look up Participant ID" button
3. Perform lookups and verify alert presence

### User Facing Changes

 - [ ] Screenshots of UI changes added to PR & Original Issue
 
### Documentation Updates
- [ ] Topline File Descriptions Added or Updated as Needed

### CODE REVIEW

**Comment by jcq** [comment]
### Updated Screenshot
Screen Shot 2020-03-06 at 16 18 34

**Comment by jcq** [comment]
@geronimoramos While this PR currently is scoped just to fix the issue of alerts showing when/where they shouldn't, I think there are some obvious improvements that could be made to the UX on `LookupParticipantIdModal`. Maybe you and @hschallhorn can decide if you'd like to include anything else in this PR and/or create another ticket for other design changes?

Things that come to mind:
- Use a "standard" two-button modal UX
  - Switch "Cancel" to read "Close"
  - Switch "Close" to be "Search"
- Better error handling
  - I implemented code that will display a red error alert if the lookup fails with an error, but currently the server doesn't actually return an error
  - Perhaps it's worth altering server response to return 4XX response if invalid CSS_ID or station ID are submitted...?
- Since the purpose of this modal seems to be to look up a participant ID for use on the page behind the modal, perhaps the results of the search could utilize the `CopyTextButton` to make the copy/paste operation easier for the user...?

**Comment by jcq** [comment]
@hschallhorn This PR currently removes not only the Redux-connected success alert from `QueueFlowModal`, but also the Redux-connected error alert. I think this conforms to what you had said about the usage of the alerts, but let me know if that is incorrect. I could always add back in the error alert if need be, though that may well still lead to some weird circumstances (much like what was noticed with the success alert in the original ticket).

File: client/app/queue/LookupParticipantIdModal.jsx [hunk 1]
```diff
@@ -44,60 +24,60 @@ class LookupParticipantIdModal extends React.Component {
...
-  search = () => {
-    let url = '';
-
-    if (this.state.css_id && this.state.station_id) {
-      url = `/user_info/represented_organizations?css_id=${this.state.css_id}&station_id=${this.state.station_id}`;
-    } else {
-      url = `/users/${this.state.selectedUser.value.id}/represented_organizations`;
-    }
+  search = async () => {
+    const { css_id: cssId, station_id: stationId, selectedUser } = this.state;
```
Comment (leikkisa) [inline client/app/queue/LookupParticipantIdModal.jsx]: Nice

File: client/app/queue/LookupParticipantIdModal.module.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.textfield {
```
Comment (leikkisa) [inline client/app/queue/LookupParticipantIdModal.module.scss]: I know you're the front-end lead so I defer to you ;-), but should we check with folks on overall app standards for handling styles? They're already done a few different ways it seems, and this introduces another.

Comment (jcq) [reply client/app/queue/LookupParticipantIdModal.module.scss]: Totally valid concern! Glamor is deprecated, so we want to move away from reliance on that. Current best practices tend to favor (S)CSS modules as a good combination of the localized scoping with the utilization of standard CSS syntax so things are easier to read and understand. This is what we have documented in our best practices doc:
https://github.com/department-of-veterans-affairs/caseflow/wiki/Styling

**Comment by geronimoramos** [comment]
@jcq thanks for the heads up! I have some ideas in the image below and shall we make a separate issue @hschallhorn ?
LookUp participant ID  modal wireframe

File: client/app/queue/LookupParticipantIdModal.module.scss (hunk 2 above)
Comment (leikkisa) [reply client/app/queue/LookupParticipantIdModal.module.scss]: Great response! And now I know :-D

File: client/app/queue/LookupParticipantIdModal.jsx [hunk 3]
```diff
@@ -110,77 +90,76 @@ class LookupParticipantIdModal extends React.Component {
...
+        >
+          <React.Fragment>
+            {alert && (
+              <div className={styles.alert}>
+                {alert === 'success' && <Alert type={alert} title="Lookup Succeeded" />}
+                {alert === 'error' && (
+                  <Alert type={alert} title="Error">
+                    There was an error processing your request. Please retry your action and contact support if errors
+                    persist.
+                  </Alert>
```
Comment (hschallhorn) [inline client/app/queue/LookupParticipantIdModal.jsx]: My only concern with this genericized error is we lose some of the validation error messages from the back end 

Before|After
---|---
Screen Shot 2020-03-11 at 12 19 39 PM|Screen Shot 2020-03-11 at 12 20 10 PM

Comment (jcq) [reply client/app/queue/LookupParticipantIdModal.jsx]: Very good point! No sense making things less clear. Just pushed an update that will allow display of the error messages like so:

Screen Shot 2020-03-12 at 16 34 24

Comment (jcq) [reply client/app/queue/LookupParticipantIdModal.jsx]: @hschallhorn Does that cover the issue that you flagged?

Comment (hschallhorn) [reply client/app/queue/LookupParticipantIdModal.jsx]: Yes! I may rest easy tonight 🙌 .

Comment (hschallhorn) [reply client/app/queue/LookupParticipantIdModal.jsx]: One last suggestion is that we pull out "There was an error processing your request...." into copy as we use it in more than one place. Would solve the linter line length warning as well

**Pull request merged**